udp/udp.go

```diff
@@ -0,0 +1,86 @@
+package udp
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/influxdb/influxdb"
+	"log"
+	"net"
+)
+
+const (
+	udpBufferSize = 65536
+)
+
+// SeriesWriter defines the interface for the destination of the data.
+type SeriesWriter interface {
+	WriteSeries(database, retentionPolicy string, points []influxdb.Point) (uint64, error)
+}
+
+// UDPServer
+type UDPServer struct {
+	writer SeriesWriter
+}
+
+// NewUDPServer returns a new instance of a UDPServer
+func NewUDPServer(w SeriesWriter) *UDPServer {
+	u := UDPServer{
+		writer: w,
+	}
+	return &u
+}
+
+// ListenAndServe binds the server to the given UDP interface.
+func (u *UDPServer) ListenAndServe(iface string) error {
+
+	addr, err := net.ResolveUDPAddr("udp", iface)
+	if err != nil {
+		log.Printf("Failed resolve UDP address %s. Error is %s", iface, err)
```

Comment:
Minor: I think the rest of our log messages are something like:

`Failed to resolve UDP address %s: %s ", iface, err)`

In other words, no need for the text "Error is...".